app/views/layouts/_flash.html.haml

```diff
@@ -0,0 +1,4 @@
+- flash.each do |name, msg|
+  - if name.eql?("notice")
+    - name = "success"
+  = content_tag :div, msg, class: "alert-" + name
```

Comment:
Doesn't this class need to be `alert alert-whatever`? http://getbootstrap.com/components/#alerts